core/src/main/java/org/axonframework/queryhandling/SimpleQueryBus.java

```diff
@@ -105,7 +112,24 @@ public SimpleQueryBus(TransactionManager transactionManager) {
     public SimpleQueryBus(MessageMonitor<? super QueryMessage<?, ?>> messageMonitor,
                           TransactionManager transactionManager,
                           QueryInvocationErrorHandler errorHandler) {
+        this(messageMonitor, NoOpMessageMonitor.INSTANCE, transactionManager, errorHandler);
+    }
+
+    /**
+     * Initialize the query bus with the given {@code messageMonitor} and given {@code errorHandler}.
```

Comment:
This javadoc body should be a bit more expanded to actually state what is given i.o. just the `messageMonitor` and `errorHandler`.